resources/views/companies/joboffers/edit.blade.php

```diff
@@ -42,8 +42,17 @@
                                    placeholder="Chief Executive Officer" value="{{ $joboffer->title or '' }}" required>
                         </div>
 
-
                         <div class="form-group">
+                            <label>Offer information type</label>
+                            <select class="form-control information_type_selector">
```

Comment:
As information_type_selecter specifies a specific field, shouldn't this be an id instead of a class?